frameworks/Scala/finagle/src/main/scala/Main.scala

```diff
@@ -13,14 +13,17 @@ import com.twitter.io.Buf
 object Main extends App {
 
   val mapper: ObjectMapper = new ObjectMapper().registerModule(DefaultScalaModule)
-  val dateFormat: DateFormat = new SimpleDateFormat("E, dd MMM yyyy HH:mm:ss z")
+
+  val dateFormat: ThreadLocal[DateFormat] = new ThreadLocal[DateFormat] {
+    override def initialValue: DateFormat = new SimpleDateFormat("E, dd MMM yyyy HH:mm:ss z")
```

Comment:
Thanks @cacoco! I updated the PR to use request API `req.date = ???` instead that uses the right date format.